ui/components/multichain/pages/permissions-page/connection-list-item.js

```diff
@@ -55,6 +51,7 @@ export const ConnectionListItem = ({ connection, onClick }) => {
       >
         {isSnap ? (
           <SnapAvatar
+            className="connection-list-item__snap-avatar"
```

Comment:
Nit: I think this can be a `data-testid` since we aren't using it for styling.

Reply:
I'll follow up on this in coming PRs, looks like the DS team is looking into adding data-testid to that SnapAvatar component.